lib/api/_loaders/_command-loader.js

```diff
@@ -34,8 +34,9 @@ class BaseCommandLoader extends BaseLoader {
     const redact = this.module.RedactParams || false;
     const alwaysAsync = this.module.alwaysAsync || false;
     const isTraceable = this.module.isTraceable;
+    const resolveParentNodeWithResult = this.module.resolveParentNodeWithResult && true;
```

Comment:
`&&` operator is wrong here (it won't do anything). After making the getter negative, we can either use `!!` as a prefix to variable, or use the [`Boolean()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean).

```suggestion
    const avoidPrematureParentNodeResolution = Boolean(this.module.avoidPrematureParentNodeResolution);
```